ethdb/database.go

```diff
@@ -10,7 +11,8 @@ import (
 	"github.com/syndtr/goleveldb/leveldb/opt"
 )
 
-var OpenFileLimit = 64
+
+var OpenFileLimit = common.MaxOpenFileLimit()
```

Comment:
Thinking about it for a bit, changing the default here is not a good idea. LevelDB will (for a big enough database) eventually open that many files, in which case none are left for other tasks. I think it should be left at 64.